rasa/core/actions/forms.py

```diff
@@ -399,12 +398,11 @@ async def validate(
         )
 
         some_slots_were_validated = any(
-            isinstance(event, SlotSet) and not event.key == REQUESTED_SLOT
+            isinstance(event, SlotSet) and event.key != REQUESTED_SLOT
             for event in validation_events
-            # Ignore `SlotSet`s  for `REQUESTED_SLOT` as that's not a slot which needs
-            # to be filled by the user.
         )
 
+
```

Comment:
Function` FormAction.validate` refactored with the following changes:

- Simplify logical expression using De Morgan identities ([de-morgan](https://docs.sourcery.ai/refactorings/de-morgan/))